codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -276,22 +277,25 @@ public class <%= schema_name %> {
                 <% end %>
 
                 <% type.required_input_fields.each do |field| %>
+                  <%= java_annotations(field) %>
                   public <%= java_input_type(field.type) %> get<%= field.classify_name %>() {
                       return <%= escape_reserved_word(field.camelize_name) %>;
                   }
 
-                  public <%= type.name %> set<%= field.classify_name %>(<%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
+                  public <%= type.name %> set<%= field.classify_name %>(<%= java_annotations(field, in_argument: true) %><%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
                       this.<%= escape_reserved_word(field.camelize_name) %> = <%= escape_reserved_word(field.camelize_name) %>;
                       return this;
                   }
                 <% end %>
                 <% type.optional_input_fields.each do |field| %>
+                  <%= java_annotations(field) %>
                   public <%= java_input_type(field.type) %> get<%= field.classify_name %>() {
                       return <%= escape_reserved_word(field.camelize_name) %>;
                   }
 
-                  public <%= type.name %> set<%= field.classify_name %>(<%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
+                  public <%= type.name %> set<%= field.classify_name %>(<%= java_annotations(field, in_argument: true) %><%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
                       this.<%= escape_reserved_word(field.camelize_name) %> = <%= escape_reserved_word(field.camelize_name) %>;
+                      this.<%= escape_reserved_word(field.name) %>Seen = true;
```

Comment:
This needs to use `field.camelize_name` without `escape_reserved_word` for consistency with the instance variable name.